### ISSUE TITLE
Imagenet Weights available?

### ISSUE THREAD

**Issue by gustavz** [miraclewkf/MobileNetV2-PyTorch#1]
Hey @miraclewkf do you have trained mobilenetv2 imagenet weights available? 
Would be nice if you could share them!
Thanks!

**Comment by mathmanu** [comment]
+1

Kindly share the top-1 accuracy that you got and if possible the trained model as well.

**Comment by holyhao** [comment]
Will you share the trained weights on Imagenet ?

**Comment by Note-Liu** [comment]
can't download the pretrained weight!!!
